rust/coverage.stats

```diff
@@ -1 +1 @@
-94.2
\ No newline at end of file
+87.8
```

Comment:
We should plan to add more tests to increase the coverage.
Could you please add this to TODO list?